src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,35 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoCard } from '../TodoCard/TodoCard';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface Props {
+  todosFromServer: Todo[];
+}
 
-    <tbody>
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">1</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
+export const TodoList: React.FC<Props> = ({ todosFromServer }) => {
+  return (
```

Comment:
when you have no logic in component and just return jsx you can delete the word `return` and just return jsx in round brackets

`}) => (`